recipe/meta.yaml

```diff
@@ -22,9 +22,13 @@ source:
     patches:
       - thirdparty-mumps-linux-configure.patch  # [linux]
 
+  # CMake-based build system for Windows
```

Comment:
There seem to be fulders for ASL and MUMPs above. Shouldn't they be removed? Packages shouldn't be vendored.

Reply:
I see that I approved that in https://github.com/conda-forge/ipopt-feedstock/pull/40. Not sure why I did....

Reply:
I opened #48 about this.

Reply:
Yes, as I think you saw later those archives are not used at all on Windows.

Reply:
Should you use `# [not win]` on the downloads of mumps and asl so that you know they aren't being brought into the windows builds?

Reply:
On this line, I marked the coinor-cmake lines with `# [win]` as they are Windows-only.